packages/web-sdk/src/transports/otlp/payload/transform/types.ts

```diff
@@ -0,0 +1,57 @@
+import type { APIEvent, TransportItem } from '@grafana/faro-core';
+
+import type { Attribute } from '../attribute';
+
+export interface ResourcePayload {
+  attributes: Attribute[];
+}
+
+export interface ScopeLog {
+  scope: { name: string; version: string };
+  logRecords: LogRecordPayload[];
+}
+
+export type LogRecordPayload =
```

Comment:
I don't think it's worthile to disambiguate to payload types like this; all the properties are generic and potentially relevant

Reply:
Hmmm thought about thsi to but added them because e. g. `LogLogRecordPayload` has a bunch of properties which `MeasurementLogRecord` doesn't have.
Also for types have different non-optional properties than others. 

Would you still put everything in one type and make all properties optional then?

Reply:
You're right it does not provide much value to disambiguate here.
Changed it.